server/package.json

```diff
@@ -19,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v10.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-upgrade-node-18",
```

Comment:
Reminder to update after https://github.com/bravetechnologycoop/brave-alert-lib/pull/56 is merged

Reply:
Done